ubyssey/widgets.py

```diff
@@ -9,7 +9,10 @@
 from dispatch.apps.events.models import Event
 
 from ubyssey.helpers import EventsHelper
-from ubyssey.zones import ArticleSidebar, HomePageSidebar, HomePageSidebarBottom
+from ubyssey.zones import (
```

Comment:
This is me being a bit obsessive, but it would be nice like this:

```python
from ubyssey.zones import (
  HomePageSidebar, HomePageSidebarBottom,
  ArticleSidebar, ArticleHorizontal
)